### ISSUE TITLE
Is this project dead ?

### ISSUE THREAD

**Issue by pedromorgan** [mdbtools/mdbtools#157]
About to fork to fix a bug, but dont seem much activity, particular lots of PR's ?

**Comment by evanmiller** [comment]
@pedromorgan The best fork in my opinion, and the one that I contribute to, is that of cyberemissary:

https://github.com/cyberemissary/mdbtools

They have made a couple of 0.8.x releases. However, I'm not sure if those releases had the blessing of @brianb.

I and many others would be interested in seeing this project properly maintained.

**Comment by dkritz** [comment]
I'm relying on this tool, so I hope it's alive! Willing to follow the consensus fork and help if I can.

**Comment by evanmiller** [comment]
Hi all,

I am currently collecting mdbtools patches in the following fork:

https://github.com/evanmiller/mdbtools

Feel free to open an issue or PR over there and I will try to bundle them up later to merge upstream.

**Comment by pedromorgan** [comment]
Know what.. ...From now on am gonna use a project with at least 2 admins=control...

So many projects just sit there and pufff.. all future code gone.....
in same namespace et all.. is the snag...
and so many forks.. which one....
its a kinda sabotage in  an indirect way... kinda..

**Comment by pedromorgan** [comment]
Perfect case example.. 

-  `Maintainer missing` = https://github.com/go-gomail/gomail/issues/104
- and then a fork which I did with ivy, and she took control.. wtf
- later upon fork `Add another maintainer for continuity reasons` = https://github.com/go-mail/mail/issues/68
- what a strange real world scenario we live in with FOSS ,,, ;-

**Comment by PMeira** [comment]
> what a strange real world scenario we live in with FOSS

@pedromorgan Agreed. For MDBTools, specifically, it is (or was, at least) included in various Linux distributions. I'd expect the companies to minimally support it. Instead, many distributions keep their patches on old/broken versions and don't really contribute upstream. It's a sad state of things.

I think I'm following 3 or 4 MDBTools forks. Kinda hard to follow what's happening... The "social" aspect of GitHub forks is cute but doesn't help in practice. On the contrary, it gets really messy.

**Comment by evanmiller** [comment]
I know another fork is annoying, but I think it's worth trying to get this project some proper maintenance. I currently maintain [libxls](https://github.com/libxls/libxls) and am willing to volunteer some time into MDB Tools. Right now I'm in the "collecting patches" phase, and hope to adopt some more modern development practices (CI, testing, fuzz-testing, etc).

I emailed @brianb about a month ago about helping with maintenance but haven't heard back. If there's still nothing happening here in the next few months I think it would be worth having a discussion about doing a harder fork (e.g. with a new project name).

**Comment by evanmiller** [comment]
Good news folks! Brian has turned over the keys to the "mdbtools" organization that @pedromorgan set up for us. I am taking over duties as maintainer and will plan an official release for this fall. More information soon.
